packages/pnpm-sync-lib/etc/pnpm-sync-lib.api.md

```diff
@@ -173,4 +183,7 @@ export function pnpmSyncGetJsonVersion(): string;
 // @beta
 export function pnpmSyncPrepareAsync(options: IPnpmSyncPrepareOptions): Promise<void>;
 
+// @beta (undocumented)
+export function writePnpmSyncFileAsync(options: ICreatePnpmSyncOptions): Promise<void>;

```

Comment:
`writePnpmSyncFileAsync` --> `pnpmUpdateSyncFileAsync`

Reply:
(these functions use `pnpm` as a *prefix* and this API may update an existing file)